### PR TITLE
Fixing JavaSpringBoot timeouts

### PR DESCRIPTION
**Pull request by Warfvf**
### What does this PR do?
* Fixes TimeOut value for running 'maven build' task
* Fixes linter issues

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

#20130 

### How to test this PR?

* Go into Eclipse repository che/tests/e2e
* Run npm i and verify that all packages install without errors
* Run tsc and verify that no errors are present
* Export following variables:
  * TS_SELENIUM_BASE_URL
  * TS_SELENIUM_MULTIUSER=true
  * NODE_TLS_REJECT_UNAUTHORIZED=0
  * TS_SELENIUM_LOG_LEVEL="DEBUG"
  * TS_SELENIUM_USERNAME
  * TS_SELENIUM_PASSWORD
* Run `USERSTORY="JavaSpringBoot" npm run test-all-devfiles`
* Test should pass with imported project.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/tests/plugins/TypescriptPlugin.spec.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ suite(`The 'TypescriptPlugin and Node-debug' tests`, async () => {
             // workaround for the issue: https://github.com/eclipse/che/issues/20067
             await debugView.clickOnDebugConfigurationDropDown();
             await debugView.clickOnDebugConfigurationItem('Add Configuration...');
-
```
Comment (ScrewTSW) [inline tests/e2e/tests/plugins/TypescriptPlugin.spec.ts:140]: Your commit message talks about HappyPath.spec.ts but this commit only lints the TypescriptPlugin.spec.ts ... hmm...

File: tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ suite(`${stack} test`, async () => {
     });
 
     suite('Validation of workspace build', async () => {
-        codeExecutionTests.runTask(buildTaskName, 420_000);
+        codeExecutionTests.runTask(buildTaskName, 1420_000);
```
Comment (ScrewTSW) [inline tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts]: With your manual testing, did the build really last over 20 minutes?

**Review by ScrewTSW — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts (hunk 2 above)
Comment (Warfvf) [reply tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts]: Last time it took 18 minutes.

Comment (ScrewTSW) [reply tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts]: AOK

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Comment by ScrewTSW** [comment]
[crw-ci-test]

**Comment by dmytro-ndp** [comment]
[crw-ci-test]

**Comment by dmytro-ndp** [comment]
[crw-ci-test]

**Pull request merged**